engine/src/main/java/org/hibernate/validator/internal/engine/ConfigurationImpl.java

```diff
@@ -400,7 +404,7 @@ public ClassLoader getExternalClassLoader() {
 
 	@Override
 	public final MessageInterpolator getDefaultMessageInterpolator() {
-		return defaultMessageInterpolator;
+		return new ResourceBundleMessageInterpolator( defaultResourceBundleLocator );
```

Comment:
I thought about this approach but I must admit I really disliked the fact that we instantiate a new `ResourceBundleMessageInterpolator` each time we call this method (and `getMessageInterpolator`). I would expect us to return the same object for each call.

Thinking about it and as I noticed `Configuration` is not supposed to be thread safe, couldn't we store the interpolators the first time the methods are called? That would solve this issue.